Makefile

```diff
@@ -29,7 +29,11 @@ install-dev-tools:  ## Installs all necessary cargo helpers
 	cargo install cargo-nextest --locked
 	cargo install cargo-binstall
 	cargo binstall cargo-risczero
+ifeq ($(shell uname -mo), Darwin x86_64)
```

Comment:
Thanks for the PR!

I believe -o is not directly POSIX for `uname`. Just tested on an x86_64 machine, -s seems to be the correct one. Could you check if -s works, and then convert -o to -s?

So it should be `uname -ms`

Reply:
> Thanks for the PR!
> 
> I believe -o is not directly POSIX for `uname`. Just tested on a Mac x86_64 machine, -o does not work | -s seems to be the correct one. Could you check if -s works, and then convert -o to -s?
> 
> So it should be `uname -ms`
> 
> Edit: [pystardust/ani-cli#416 (comment)](https://github.com/pystardust/ani-cli/issues/416#issuecomment-1023607443)

Yes, thank you a lot

Reply:
Great